Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -2348,6 +2348,14 @@ private void ProcessScreenShot(Act act, GingerRunner.eActionExecutorType ActionE
             }
         }
 
+        private void SaveArtifactsScreenshot(Act act, string screenshot)
+        {
+            act.ScreenShots.Add(screenshot);
+            if (act.GetType() == typeof(ActVisualTesting))
```

Comment:
can't we catch it from the ActVisualTesting or from the Drivers somehow instead placing the IF it in generic place

Reply:
Done